parquet/tests/variant_integration.rs

```diff
@@ -274,13 +291,11 @@ impl VariantTestCase {
 
         let variant_data = self.load_variants();
         let variant_array = self.load_parquet();
+        let variant_array = unshred_variant(&variant_array).unwrap();
```

Comment:
👍

Reply:
Some context might help: 
```suggestion
        // The expected values are provided as unshredded variants, so
        // unshred the array here prior to comparison
        let variant_array = unshred_variant(&variant_array).unwrap();
```